config/initializers/assets.rb

```diff
@@ -5,6 +5,7 @@
 
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
👍  I'll update the README.

Primer is only [distributed as an NPM module](https://github.com/primer/primer-css#install). The only time anyone will need to touch node is to update it.